src/gawronski-bartosz/Fsd.Bartek.Cs/Fsd.Bartek.Ex4.Web/Controllers/ProductsController.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Fsd.Bartek.Ex4.Data.Entities;
+using Fsd.Bartek.Ex4.Services.Services;
+using Fsd.Bartek.Ex4.Web.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+namespace Fsd.Bartek.Ex4.Web.Controllers
+{
+    public class ProductsController : Controller
+    {
+        private IProductsService _productService;
+        
+        public ProductsController()
+        {
+            _productService = new ProductsService();
+        }
+
+        public ActionResult ProductsList()
+        {
+            string page = Request.QueryString["page"];
```

Comment:
nie trzeba tego wyciągać z tablicy - można to dostać jako parametry metody ProductsList()